test/fixtures/test-code-src-altoroj/AccountAPI.java

```diff
@@ -0,0 +1,237 @@
+package com.ibm.security.appscan.altoromutual.api;
```

Comment:
Can we assign the relevant team as CODEOWNERS for these new fixtures?

Reply:
understand, I didnt see the test fixtures for others separated in the code owners. all code ones are under: /test/fixtures/test-code*